### PR TITLE
More docs

### PR DESCRIPTION
**Pull request by koenvo**
This PR contains docs for event part of kloppy. All domain models for events, common and pitch are covered.

### CODE REVIEW

File: docs/api/helpers.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+::: kloppy.domain.services.state_builder.add_state
+
+::: kloppy.helpers.to_pandas
+
+::: kloppy.helpers.transform
```
Comment (bdagnino) [inline docs/api/helpers.md:5]: @koenvo it seems like helpers.md is not included in _mkdocs.yml_, is that intentional?

**Comment by bdagnino** [comment]
@koenvo reviewed! Great work documenting the API. I quite like how docstrings and mkdocs work together here. Except a comment regarding _helpers.md_ not being included in the documentation looks ready to merge!

File: docs/api/helpers.md (hunk 1 above)
Comment (koenvo) [reply docs/api/helpers.md]: Was intentional but confusing :-)

Changed it a bit

**Comment by koenvo** [comment]
> @koenvo reviewed! Great work documenting the API. I quite like how docstrings and mkdocs work together here. Except a comment regarding _helpers.md_ not being included in the documentation looks ready to merge!

Thanks! The only thing I still want to look into is loading everything from a single module and filter. The filter wasn't working as expected ( https://pawamoy.github.io/pytkdocs/#configuration ); it always showed all members..

**Pull request merged**